src/include/partitioning/custom_part_check_generated.rb

```diff
@@ -294,6 +299,20 @@ def check_created_partition_table(targetMap, installation)
         ok = false if !Popup.YesNo(message)
       end
 
+      # A PReP/CHRP partition is not supposed to be mounted. So if we find any
+      # other /boot partition, we should warn the user.
+      if boot_found && Partitions.PrepBoot && !boot_mount_point.empty? && !Arch.board_iseries && installation || show_all_popups
```

Comment:
@imobach Then I'd probably put it out in a separate function `boot_partition_needed?` - I know it's now a one-liner, but in this one-liner, you could document it and reuse it from other places. That's DRY and makes the code readable. It's just a hint...